### PR TITLE
Updated links

### PR DESCRIPTION
**Pull request by mokabiru**
The previous pull request had broken links. Fixed in this PR.

### CODE REVIEW

**Comment by rajmera3** [comment]
@mokabiru Looks like you have some merge conflicts. Can you rebase and fix those?

File: README.md [hunk 1]
```diff
@@ -70,12 +70,11 @@ The Apache Spark Connector for SQL Server and Azure SQL is based on the Spark Da
...
 To include the connector in your projects download this repository and build the jar using SBT.
 
-### Write to a new SQL Table
-#### Important: using the `overwrite` mode will first DROP the table if it already exists in the database. Please use this option with due care to avoid unexpected data loss!
+### Write to SQL Table
 ```python
 server_name = "jdbc:sqlserver://{SERVER_ADDR}"
 database_name = "database_name"
-url = server_name + ";" + "databaseName=" + database_name + ";"
+url = servername + ";" + "database_name=" + dbname + ";"
```
Comment (rajmera3) [inline README.md]: Might be mistaken but I think databaseName is a needed to properly build the JDBC string
https://docs.microsoft.com/en-us/sql/connect/jdbc/building-the-connection-url?view=sql-server-ver15

@mokabiru can you double check this line change?

File: README.md [hunk 2]
```diff
@@ -109,12 +108,6 @@ except ValueError as error :
     print("Connector write failed", error)
 ```
 
-### Specifying the isolation level
```
Comment (rajmera3) [inline README.md]: Why is this  being deleted? Looks like you might not have pulled Arvind's changes? He made a PR adding and updating information on the readME

Comment (mokabiru) [reply README.md]: @rajmera3 Apologies my file sync on OneDrive messed it up. My last commit (commit a7cf3195274f2a790c000485b74d3ae5141c22e7) has the updated README from previous commits including Arvind's.

File: README.md (hunk 1 above)
Comment (mokabiru) [reply README.md]: @rajmera3 Apologies my file sync on OneDrive messed it up. My last commit (commit a7cf3195274f2a790c000485b74d3ae5141c22e7) has the updated README from previous commits including Arvind's.

Comment (mokabiru) [reply README.md]: Commit Link: https://github.com/microsoft/sql-spark-connector/pull/12/commits/a7cf3195274f2a790c000485b74d3ae5141c22e7

File: samples/Databricks-AzureSQL/README.md [hunk 3]
```diff
@@ -1,23 +1,28 @@
...
+**Apache Spark Connector for SQL Server and Azure SQL**
 
-The connector can either be downloaded from the [azure-sqldb-spark GitHub repo](https://github.com/Azure/azure-sqldb-spark/tree/master/releases/azure-sqldb-spark-1.0.0) or imported into Azure Databricks using the Maven coordinate: `com.microsoft.azure:azure-sqldb-spark:1.0.2`
+The [Spark connector for SQL Server and Azure SQL](https://docs.microsoft.com/en-us/azure/azure-sql/database/spark-connector) enables databases in Azure SQL Database, Azure SQL Managed Instance, and SQL Server to act as the input data source or output data sink for Spark jobs. The Spark connector is a high-performance connector that enables you to use  transactional data in big data analytics and persists results for ad-hoc queries or reporting.
```
Comment (rajmera3) [inline samples/Databricks-AzureSQL/README.md:12]: Sorry one last change -  this docs link is to the Azure SQL Spark connector. Note that this one is different from this project. We do not have a docs page for this connector as of now other than a page in the SQL Server Big Data Clusters docs

Comment (mokabiru) [reply samples/Databricks-AzureSQL/README.md]: Done and committed

Comment (rajmera3) [reply samples/Databricks-AzureSQL/README.md]: Thanks!

**Pull request merged**